WatchApp Extension/View Models/OnOffSelectionViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  OnOffSelectionViewModel.swift
+//  WatchApp Extension
+//
+//  Created by Anna Quinlan on 8/20/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+enum SelectedButton {
+    case on
+    case off
+    case none
```

Comment:
[nit] Is there an actual use case where `none` is valid?

Reply:
If so, then should it be handled in the `body` of the view?

Reply:
This was for backwards-compatibility (in case someone wanted to have an on/off sheet without the coloring), but since it could be easily added in later and there isn't a super compelling usecase, I removed it